airbyte-integrations/connectors/destination-mssql-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLCheckTest.kt

```diff
@@ -3,11 +3,19 @@ package io.airbyte.integrations.destination.mssql.v2
 import io.airbyte.cdk.load.check.CheckIntegrationTest
 import io.airbyte.cdk.load.check.CheckTestConfig
 import io.airbyte.integrations.destination.mssql.v2.config.MSSQLSpecification
+import java.util.regex.Pattern
 
-class MSSQLCheckTest : CheckIntegrationTest<MSSQLSpecification>(
-    successConfigFilenames = listOf(
-        CheckTestConfig(MSSQLTestConfigUtil.getConfigPath("check/valid.json")),
-    ),
-    failConfigFilenamesAndFailureReasons = emptyMap()
-) {
-}
+class MSSQLCheckTest :
+    CheckIntegrationTest<MSSQLSpecification>(
+        successConfigFilenames =
+            listOf(
+                CheckTestConfig(MSSQLTestConfigUtil.getConfigPath("check/valid.json")),
+                CheckTestConfig(MSSQLTestConfigUtil.getConfigPath("check/valid-ssl-trust.json")),
+            ),
+        failConfigFilenamesAndFailureReasons =
+            mapOf(
+                CheckTestConfig(
+                    MSSQLTestConfigUtil.getConfigPath("check/fail-internal-schema-invalid.json")
+                ) to Pattern.compile("\"iamnotthere\" either does not exist"),
```

Comment:
Nit:  you can go `"some string".toPattern()` in Kotlin instead of `Pattern.compile`.